lib/task_helpers.rb

```diff
@@ -60,6 +60,11 @@ def self.create_index(name=nil)
       end
 
       Elasticsearch::Model.client.indices.create(index: name, body: {mappings: mappings})
+
+      [Comment, CommentThread].each do |model|
+        model.put_search_index_mapping(name)
+      end
```

Comment:
FYI: @maxrothman @dianakhuang: I'm unclear at this point whether this code is ok or not.  If @clintonb isn't clear, it is going to require a little digging.  And this will be high priority, because I want to make sure that master does not have any defects.